src/Microsoft.Framework.ApplicationHost2/project.json

```diff
@@ -58,7 +56,8 @@
                 "System.Linq": "4.0.0-beta-*",
                 "System.Resources.ResourceManager": "4.0.0-beta-*",
                 "System.Threading": "4.0.10-beta-*",
-                "System.Text.RegularExpressions": "4.0.10-beta-*"
+                "System.Text.RegularExpressions": "4.0.10-beta-*",
```

Comment:
Again, used in FrameworkReferenceResolver. Sure it *should* be hard-coded, but it isn't now and that would be a separate work item IMHO.